src/main/resources/solrconfig.xml

```diff
@@ -735,6 +735,7 @@
     <lst name="defaults">
       <str name="echoParams">explicit</str>
       <int name="rows">10</int>
+      <str name="df">text_multi</str>
```

Comment:
```suggestion
      <str name="df">text_multi</str><!-- PRL -->
```

One difficulty I found in understanding the Solr 4.x config files was that they lacked comments indicating which changes were necessary (which is one reason why I initially glossed over this one). Perhaps I'm being a bit overly cautious, but this seems like a good idea to me.